src/pybind/mgr/dashboard/monitors.html

```diff
@@ -0,0 +1,98 @@
+
+{% extends "base.html" %}
+
+{% block content %}
+
+
+<script>
+        $(document).ready(function(){
+            // Pre-populated initial data at page load
+            var content_data = {{ content_data }};
+	     var refresh = function() {
+	         $.get("{{ url_prefix }}/monitors_data", function(data) {
+	             _.extend(content_data, data);
+	             $('.inlinesparkline').sparkline();
+	             setTimeout(refresh, 5000);
+	         });
+	     };	
+	     setTimeout(refresh, 5000);
+	
+	     rivets.formatters.sparkline_data = function(time_series) {
+	         result = "";
+	         for (var i = 1; i < time_series.length; ++i) {
+		     var y = time_series[i][1];
+		     result += (y + ",");		  
+	         }
+	         return result;
+	     }
+             rivets.bind($("#content"), content_data);
+             $(".inlinesparkline").sparkline();
+         });
+</script>
+
+<!-- Page Header -->
+<section class="content-header">
+    <h1 style="font-weight:bold">
+      Monitors
+    </h1>
+</section>
+	
+<!-- Main content -->
+<section class="content text-center">
+    <div class="row">
+        <div class="col-md-4">
+	    <div class="box text-left" style="font-size:13.4px">
+                <table class="table table-bordered">
+		    <tr><td><p><span style="color:yellow; font-weight:bold">Cluster ID: </span></td><td>{mon_status.monmap.fsid}</p></td></tr>
+		    <tr><td><p><span style="color:yellow; font-weight:bold">monmap modified: </span></td><td> {mon_status.monmap.modified}</p></td></tr>
+		    <tr><td><p><span style="color:yellow; font-weight:bold">monmap epoch: </span></td><td> {mon_status.monmap.epoch}</p></td></tr>
+		    <tr><td><p><span style="color:yellow; font-weight:bold">quorum con: </span></td><td> {mon_status.features.quorum_con}</p></td></tr>
+		    <tr><td><p><span style="color:yellow; font-weight:bold">required con: </span></td><td> {mon_status.features.required_con}</p></td></tr>
```

Comment:
While I think `quorum_con` and `required_con` features are nice values to know, I think the integer values have little value when exposed by themselves to the user on the dashboard. It would be nice to actually have the features being used instead.

It may just so be that you do not have a good way to obtain those; if that is the case, please file a ticket in the tracker mentioning just that, assign it to me, link this PR in the ticket, and I'll address that (and then we can amend this feature to include those features at a later date).